index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar">
+        <a href="#home" class="logo">John Doe</a>
+       <nav class="harmburger flex">
+        <span class="line"></span>
+        <span class="line"></span>
+        <span class="line"></span>
+       </nav>
+      </nav>
+      <section class="headline flex" id="home">
+        <h1>Hey There.
+            I’m David
+        </h1>
+        <h3>I’am a Software Developer</h3>
+        <div class="headline-details">
+            <p class="lead">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <ul class="icons flex">
+                <li><img class="github" src="asset/github.png" alt="github"/></li>
+                <li>
+                    <span class="in flex">
+                        <span class="i flex">
+                            <img src="asset/dot.png" alt="dot">
+                            <img src="asset/i.png" alt="i">
+                        </span>
+                        <img class="in" src="asset/magnet.png" alt="n"/>
+                    </span>
+                </li>
+                <li><img class="rabbid" src="asset/rabbid.png" alt="rabbid"/></li>
+                <li><img class="twitter" src="asset/twitter.png" alt="twitter"/></li>
+                <li><img class="m" src="asset/m.png" alt="github"/></li>
+            </ul>
+        </div>
+        <img src="asset/Group.png" alt="Group" class="i-image">
+      </section>
+    </header>

```

Comment:
Kindly, please notice the followings:
- you cannot have a `nav` tag inside a `nav` tag; please keep in mind that you need in your `header` the logo and the navigation and not nested navigation into another navigation;
- a `section` tag cannot be nested inside a `nav` tag; kindly, take it outside of navigation and only make it a sibling of the navigation
- your indentation and tags display should look like this:
`<body>`
`--<header>`
`----<a>logo</a>`
`----<nav>hamburger button</nav>`
`--</header>`
`--<section class="headline">`
`----headline content`
`--</section>`
`</body>`

- [ ] Please, pay attention to these aspects and correct them;

Reply:
What chances do I have that after fixing this like so, another reviewer won’t jump in to review my code and find another fault.

I first had just a div wrapping the hamburger but my first reviewer flagged it as wrong, although I know it is but I had to do what he wanted to pass and here you are flagging his correction as an error.

i hope you get what I’m trying to say here

Reply:
pls sir, can I use a button tag instead of the nav tag wrapping the hamburger